src/components/AvatarWithImagePicker.js

```diff
@@ -12,7 +12,7 @@ import themeColors from '../styles/themes/default';
 import AttachmentPicker from './AttachmentPicker';
 import AvatarCropModal from './AvatarCropModal/AvatarCropModal';
 import OfflineWithFeedback from './OfflineWithFeedback';
-import withLocalize, {withLocalizePropTypes} from './withLocalize';
+import useLocalize from '../hooks/useLocalize';
 import variables from '../styles/variables';
 import CONST from '../CONST';
 import SpinningIndicatorAnimation from '../styles/animation/SpinningIndicatorAnimation';
```

Comment:
i leave it as it is as your comment is unclear and I was already asked to do as it is now

Reply:
@keisyrzk You shouldn't resolve comments because something is 'unclear'. Instead please ask a question or just ask to make the comment more clear.

The thing is that `SpinningIndicatorAnimation` in this component is not really used (it's used only in lifecycle methods):
```ts
 const animation = useRef(new SpinningIndicatorAnimation()).current;

    useEffect(() => {
        if (isUploading) {
            animation.start();
        }

        return () => {
            animation.stop();
        };
    }, [isUploading]);
```

> I was already asked to do as it is now

What do you mean? Who did it and what's the reasoning behind it?

Reply:
ok i misunderstood you

Reply:
No problem 😄